Products/ZenEvents/zentrap.py

```diff
@@ -760,6 +774,12 @@ class TrapDaemon(CollectorDaemon):
 
     _frameworkFactoryName = "nosip"
 
+    def __init__(self, *args, **kwargs):
+        self._trapFilter = TrapFilter()
+        zope.component.provideUtility(self._trapFilter, ICollectorEventTransformer)
+        super(TrapDaemon, self).__init__(*args, **kwargs)
+        self._trapFilter.initialize()
```

Comment:
The implementation in 36d5627 only half works - no error event is displayed in RM. Not sure what I'm missing